### PR TITLE
 Fix Issue#28

### PR DESCRIPTION
**Pull request by MOHDNEHALKHAN**
@PiyushKalyanpy, I have added the right-padding side of the Explore Courses Button 
Before Changes - 
Screenshot 2023-05-26 000156
After Changes -
Screenshot 2023-05-26 000031

### CODE REVIEW

File: Components/HomeSidebar.tsx [hunk 1]
```diff
@@ -20,16 +19,20 @@ const HomeSidebar = ({ pageNumber = 1 }: any) => {
   };
 
   return (
-    <div className="fixed top-0 flex  flex-col  p-4 w-fit h-screen justify-between ">
-      {/* <LogoWithName width={40} height={40} /> */}
-      <div className="flex flex-col space-y-4">
+    <div className="absolute top-0 flex  flex-col  p-4 justify-between w-[304px] h-screen left-[1px] bg-gradient-to-b from-[#e2e9f1] to-white">
+      {/* Top Part  */}
```
Comment (lalitmee) [inline Components/HomeSidebar.tsx:23]: No need of the comments in HTML part. Please remove all the comments from HTML in this file.

File: Components/HeroSection.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ const HeroSection = () => {
               />
               <div
                 onClick={() => router.push("/courses")}
-                className="flex flex-row items-center gap-4 p-4 text-white bg-black border-2 border-black rounded-full h-fit cursor-pointer dark:bg-zinc-50"
+                className="flex flex-row items-center gap-4 p-4 text-white bg-black border-2 border-black rounded-full h-fit cursor-pointer dark:bg-zinc-50 pr-7"
```
Comment (lalitmee) [inline Components/HeroSection.tsx:77]: don't add padding to make it align in the center.

Just use flex properties as its already using `flex`

File: package-lock.json [hunk 3]
```diff
@@ -5438,7 +5438,8 @@
     "@firebase/auth-types": {
       "version": "0.12.0",
       "resolved": "https://registry.npmjs.org/@firebase/auth-types/-/auth-types-0.12.0.tgz",
-      "integrity": "sha512-pPwaZt+SPOshK8xNoiQlK5XIrS97kFYc3Rc7xmy373QsOJ9MmqXxLaYssP5Kcds4wd2qK//amx/c+A8O2fVeZA=="
```
Comment (lalitmee) [inline package-lock.json]: don't push `package-lock.json` if you haven't installed a new package in `package.json`

File: Components/HomeSidebar.tsx [hunk 4]
```diff
@@ -20,16 +19,20 @@ const HomeSidebar = ({ pageNumber = 1 }: any) => {
...
+      <LogoWithName width={40} height={40} />
+
+      {/* Center Part  */}
+      <div className="flex flex-col items-start gap-[24px] p-0 w-[304px] h-[360px] top-[207px] left-[calc(50%-152px)] absolute">
```
Comment (lalitmee) [inline Components/HomeSidebar.tsx:27]: What are these changes? There is nothing about these in the PR description. Please, always try to explain a little bit about your changes in the PR description.

File: Components/LogoWithName.tsx [hunk 5]
```diff
@@ -1,9 +1,9 @@
 import Image from "next/image";
 const LogoWithName = ({width, height} : any) => {
   return (
-    <div className="flex flex-row items-center">
+    <div className="flex flex-row items-center p-0 gap-6 w-[230px] h-[49px] top-[43px] left-[37px] justify-center absolute">
```
Comment (lalitmee) [inline Components/LogoWithName.tsx:4]: Can you please add `before` and `after` screenshots of all the UI changes?

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by PiyushKalyanpy** [comment]
The changes requested were not done, so we acre closing this PR, please reopen the PR once you made the changes.

**Pull request closed without merging**